modules/S22_php_check.sh

```diff
@@ -46,11 +54,11 @@ S22_php_check()
         (( S22_PHP_VULNS="$S22_PHP_VULNS"+"$VULNS" ))
       done < "$TMP_DIR"/S22_VULNS.tmp
     fi
-
+    s22_check_php_ini
     print_output ""
     print_output "[+] Found ""$ORANGE""$S22_PHP_VULNS"" issues""$GREEN"" in ""$ORANGE""$S22_PHP_SCRIPTS""$GREEN"" php files.""$NC""\\n"
     write_log ""
-    write_log "[*] Statistics:$S22_PHP_VULNS:$S22_PHP_SCRIPTS"
+    write_log "[*] Statistics:$S22_PHP_VULNS:$S22_PHP_SCRIPTS:$S22_PHP_INI_LIMIT_EXCEEDED:$S22_PHP_INI_FAILURE:$S22_PHP_INI_WARNINGS" >> "$LOG_FILE"
```

Comment:
I can't find the code where you are going to populate the variables. So, these will always be zero.

Reply:
See row: 124, 136, 137

Reply:
there is naming issue S22_PHP_INI_LIMIT_EXCEEDED vs PHP_INI_LIMIT_EXCEEDED. Currently all the values will be zero.